boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig

```diff
@@ -29,6 +29,13 @@ CONFIG_MEMC=y
 # enable SDRAM
 CONFIG_MEMC_STM32_SDRAM=y
 
+# enable spi
+CONFIG_SPI=y
+CONFIG_SPI_STM32=y
```

Comment:
This is not required, when `CONFIG_SPI=y` STM32 SPI is auto-selected.

Reply:
Got you! Ok so for all these subsystems I always only need to add CONFIG_SPI or MEMC etc and never soc level selection? Is that the general rule?

Reply:
Yes

Reply:
SPI should be enabled in the sample board overlay